tsconfig.json

```diff
@@ -7,13 +8,15 @@
         "sourceMap": true,
         "target": "ES6",
         "declaration": true,
-        "moduleResolution": "node"
+        "moduleResolution": "node",
+        "outDir": "lib"
     },
     "include": [
         "./declarations.d.ts",
         "./src/**/*.ts",
         "./script/**/*.ts",
-        "./test/**/*.ts"
+        "./test/**/*.ts",
+        "src"
```

Comment:
"src" should not be necessary since we already have `"./src/**/*.ts"`. Use only 1 of them.

Reply:
I was having trouble with the auto-compile and thought that was the problem, turns out it wasn't.  Removed it.